docs/variants/protectli_vp2420/building-manual.md

```diff
@@ -0,0 +1,37 @@
+# Building manual
+
+## Intro
+
+This document describes the procedure for compiling coreboot for Protectli
+VP2420.
+
+## Requirements
+
+- Docker
+    + follow [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu/)
+    + follow [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/)
+- Git
+
+## Build Dasharo BIOS firmware
+
+1. Clone the coreboot repository:
+
+    ```bash
+    git clone https://github.com/Dasharo/coreboot.git \
+    -b protectli_vault_glk/release
+    ```
```

Comment:
This is wrong. Definitely not protecli_vault_glk/release.  Please have a look how this is done here: https://github.com/Dasharo/docs/blob/master/docs/variants/protectli_vp46xx/building-manual.md (should be mostly the same here)

Reply:
Fixed the procedure. But this also made me think, is there any reason why steps 3. and 4. from the link you gave isn't done in the build script? We could just clone there the repository with blobs that we have.

Reply:
No we can't just publish blobs like that...